Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/dotnet/core/runtime:2.2-alpine
+
+COPY src/Analyzer/bin/Release/netcoreapp2.2/publish/*.dll /opt/docker/bin/
```

Comment:
this is dependent from operations done on the host machine IIUC, can we use multi-stage build here?

Reply:
No, this just depends on the framework target, and here, we just use one framework target, which is `netcoreapp2.2`. The `.dll` file is CIL bytecode, so it's host independent. You just need to make sure you have the same runtime installed.

Reply:
this doesn't come with the repository itself.
Should be done in a multi-stage IMHO

Reply:
Done